### ISSUE TITLE
[question] `static` parsed as constref

### ISSUE THREAD

**Issue by evilebottnawi** [glayzzle/php-parser#183]
```php

class A {
    public static function test() {
        static::who();
    }
}

```

But `self/parent` as `identifier`. Looks some misleading. Can you describe why? Or maybe it is bug in parser?

**Comment by ichiriac** [comment]
not sure why, I need to check the lexer but it may be because self & parent are keywords and static my be a T_STRING like any class name.

Anyway I'm agree with you that it's not consistent and should be fixed.

**Comment by ichiriac** [comment]
its now fixed, need to release it

**Comment by ichiriac** [comment]
released under `php-parser@3.0.0-prerelease.6`
